### PR TITLE
Translator Text API V3

### PR DESCRIPTION
**Pull request by yangyuan**
This PR is for replacing below PRs:
* https://github.com/Azure/azure-rest-api-specs/pull/3943
* https://github.com/Azure/azure-rest-api-specs/pull/3705
* https://github.com/Azure/azure-rest-api-specs/pull/3707

As Kelly has left, I will help to finish the PR.
We have reviewed the API and swagger with Azure API review board.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by yangyuan** [comment]
not sure why go is failing. @sarangan12 can you help to take a look

File: specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json [hunk 1]
```diff
@@ -0,0 +1,1118 @@
...
...ope`, `westus`, and `westus2`.\n\nRegion is required for the all-in-one Text API subscription.\n\n\n# Errors\n\nA standard error response is a JSON object with name/value pair named `error`. The value is also a JSON object with properties:\n  * `code`: A server-defined error code.\n  * `message`: A string giving a human-readable representation of the error.\n\nFor example, a customer with a free trial subscription receives the following error once the free quota is exhausted:\n\n```json\n{\n  \"error\": {\n  \"code\":403000,\n  \"message\":\"The subscription has exceeded its free quota.\"\n  }\n}\n```\n# Enter your subcription keys to try out Microsoft Translator.\nSelect the `Authorize` button and enter your Microsoft Translator subscription key, OR your `all in one Cognitive Services` subscription key. If you are using the all in one Cognitive Services key you will need to also enter your subscription region.\n## Available regions are:\n\n  `australiaeast`, `brazilsouth`, `canadacentral`, `centralindia`, `centraluseuap`, `eastasia`, `eastus`, `eastus2`, `japaneast`, `northeurope`, `southcentralus`, `southeastasia`, `uksouth`, `westcentralus`, `westeurope`, `westus`, `westus2`.\n"
```
Comment (nschonni) [inline specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json]: subcription -> subscription

File: specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json [hunk 2]
```diff
@@ -0,0 +1,1118 @@
...
+              }
+            }
+          }
+        }
+      }
+    },
+    
+    "BreakSentenceResult": {
+      "type": "array",
+      "description": "Example of a successful breaksentence request",
```
Comment (nschonni) [inline specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json]: breaksentence -> break sentence or BreakSentence

File: specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json [hunk 3]
```diff
@@ -0,0 +1,1118 @@
...
+        "properties": {
+          "text": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "BreakSentenceTextInput": {
+      "type": "object",
+      "description": "Text needed for breaksentence request ",
```
Comment (nschonni) [inline specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json]: breaksentence -> break sentence or BreakSentence

File: specification/cognitiveservices/data-plane/TranslatorText/readme.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-go
+  - repo: azure-sdk-for-nodejs
+ 
+  
+    after_scripts:
```
Comment (sarangan12) [inline specification/cognitiveservices/data-plane/TranslatorText/readme.md]: This is required only if you want Ruby SDK. Either remove after_scripts section or add azure-sdk-for-ruby above it

**Review by sarangan12 — CHANGES_REQUESTED** [review]

File: specification/cognitiveservices/data-plane/TranslatorText/readme.md (hunk 4 above)
Comment (yangyuan) [reply specification/cognitiveservices/data-plane/TranslatorText/readme.md]: `after_scripts` removed~

**Comment by sarangan12** [comment]
@jhendrixMSFT Can you please take a look why Go generation is failing? The output says there is no input files provided. But, I am able to see the files for go. Could you please check? TIA

File: specification/cognitiveservices/data-plane/TranslatorText/readme.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: translatortext
+  clear-output-folder: true
+```
+
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: release_3_0 and go
```
Comment (jhendrixMSFT) [inline specification/cognitiveservices/data-plane/TranslatorText/readme.md]: The tag name is wrong it should be `release_3_0`

Comment (yangyuan) [reply specification/cognitiveservices/data-plane/TranslatorText/readme.md]: I removed ` and go` see if it works

**Comment by yangyuan** [comment]
@sarangan12 all pass now :)

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**